docs/proposals/auto-install-roles.md

```diff
@@ -0,0 +1,106 @@
+# Auto Install Ansible roles
+
+*Author*: Will Thames <@willthames>
+
+*Date*: 19/02/2016
+
+## Motivation
+
+To use the latest (or even a specific) version of a playbook with the
+appropriate roles, the following steps are typically required:
+
+```
+git pull upstream branch
+ansible-galaxy install -r path/to/rolesfile.yml -p path/to/rolesdir -f
+ansible-playbook run-the-playbook.yml
+```
+
+The most likely step in this process to be forgotten is the middle step. While
+we can improve processes and documentation to try and ensure that this step is
+not skipped, we can improve ansible-playbook so that the step is not required.
+
+## Approaches
+
+### Approach 1: Specify rolesfile and rolesdir in playbook
+
+Provide new `rolesdir` and `rolesfile` keywords:
```

Comment:
Lines 40 and 41 discuss the configuration options. 

`roles_path` is fine for this purpose as long as `roles_path` can be relative to the playbook being run. I'm not sure if that's the case right now.